### DISCUSSION TITLE
aapt tool failed: Error: No resource found that matches the given name (at 'resource' with value '@drawable/ic_stat_distriqt').

### DISCUSSION THREAD

**Discussion by mc3dev** [airsdk/Adobe-Runtime-Support#1340] [Q&A]
Hi, I have since upgraded to the latest AIR SDK 33.1.1.633, but I am still seeing the same issue and cannot pass the error to publish an Android .apk file. Can you please check this and any ideas as to why it isn't finding the resources - https://github.com/distriqt/ANE-PushNotifications/issues/512

Thanks. Really appreciated it.

**Comment by ajwfrost** [comment]
Hi @mc3dev  - can you please edit the file "adt.cfg" that should be in your SDK's "lib" folder, and uncomment the line:
`DebugOut=true`
Then if you run it again, you should find a log file being generated in your home folder (c:\users\username\adt.log): if you can share that, it will hopefully have some information about the internal mechanisms and we can see why it's not picking up the resource folder..

thanks

**Reply by mc3dev** [reply]
[adt.log](https://github.com/airsdk/Adobe-Runtime-Support/files/7509894/adt.log)
@ajwfrost Attached is the adt.log file saved based on your instructions. Please check and let me know if you can see the issues. Thank you very much for your help.

**Reply by ajwfrost** [reply]
Okay thanks -> so the aapt compilation command-line is starting with (simplifying it a bit):
```
AIR33.1.1.633\lib\android\bin\aapt.exe package -z -u -J TmpFolder
-M TmpFolder\AndroidManifest.xml
-S TmpFolder\app_entry_res
-S TmpFolder\captive_runtime_res
-m
--extra-packages  .....
```

But if this has found the extra `resdir` entry, it should have been inserted prior to `app_entry_res` and after the manifest.

I am wondering whether there's a problem with the resolution of the folder within the `resdir` entry, perhaps because this is coming via an IDE. It's interesting to note that the original arguments into ADT are being passed with the full path names in them so that e.g. when we pick up the application descriptor file or the SWF file these are being provided as fully qualified paths. Within ADT, we just read whatever is provided on the command-line (or in that descriptor file) and allow Java to resolve the path based on the current working folder...

So: to get past this in the short term, please change the `resdir` entry to be the full folder name/path starting with "C:/" (best to use forward-slashes for Java..) and we'll check this when running in Animate to see if we need to adjust how we pick up the folders...

thanks
